drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/RightPanelViewImpl.html

```diff
@@ -42,7 +42,7 @@
         <!-- content for the test editor tab -->
         <div id="kieTestEditorTabContent"
              class="tab-pane fade in active"
-             aria-labelledby="Test Editor">
+             aria-labelledby="Test Editor" style="padding: 1em">
             <br/>
             <p>To create a test template, define "Given" and "Expect" columns by entering an expression or use the expression editor below.</p>
```

Comment:
Change line number 47 to >> To create a test template, define **the** "Given" and "Expect" columns by entering an expression or **using** the expression editor below.